internal/clients/nightfall/nightfall.go

```diff
@@ -278,3 +284,34 @@ func (n *Client) ReviewDiff(
 
 	return comments, nil
 }
+
+func filterFileDiffs(fileDiffs []*diffreviewer.FileDiff, fileIncludeList, fileExcludeList []string) []*diffreviewer.FileDiff {
+	filteredFileDiffs := fileDiffs
+	if fileIncludeList != nil && len(fileIncludeList) > 0 {
+		filteredFileDiffs = filterByFilePath(filteredFileDiffs, fileIncludeList, true)
+	}
+	if fileExcludeList != nil && len(fileExcludeList) > 0 {
+		filteredFileDiffs = filterByFilePath(filteredFileDiffs, fileExcludeList, false)
+	}
+	return filteredFileDiffs
+}
+
+func filterByFilePath(fileDiffs []*diffreviewer.FileDiff, regexPatterns []string, include bool) []*diffreviewer.FileDiff {
+	filteredFileDiffs := []*diffreviewer.FileDiff{}
```

Comment:
let's make this slice (using `make`) and specifying a capacity hint (maybe jus the len of `fileDiffs`) so that we hopefully never have to reallocate space for the array
e.g. `make([]*diffreviewer.FileDiff, 0, len(fileDiffs)`

https://github.com/uber-go/guide/blob/master/style.md#specifying-slice-capacity